frontend/index.html

```diff
@@ -134,16 +167,18 @@ <h2 class="subtitle">Popular categories</h2>
     <!-- Trending now -->
     <section>
         <h2 class="subtitle">Trending now</h2>
-        <div id="trending" class="trending grid" />
+        <div id="trending" class="trending grid">
         <template id="trending-product-template">
-            <a href="trending">
-                <div>
+            <a href="#trending" class="trending-div-product-grid">
```

Comment:
If possible, I'd try to not let the anchor element wrap the "Add to cart" button, since clicking that button would invoke both the link and button events, also adding `trending` to the url. 

<img width="851" alt="AddToCartClicked" src="https://github.com/MinhazH99/orders101/assets/1670145/6cde7663-28b3-4feb-af14-32cb5165ef7d">

Since there's no product page yet, perhaps we could link to `#` for now?

Reply:
Button was moved out of the anchor tag. Clicking on trending items also lead to user going to the product page